rotkehlchen/constants/assets.py

```diff
@@ -277,7 +277,7 @@
 A_GLM = Asset('eip155:1/erc20:0x7DD9c5Cba05E151C895FDe1CF355C9A1D5DA6429')
 
 A_WETH_OPT = Asset('eip155:10/erc20:0x4200000000000000000000000000000000000006')
-A_WETH_ARB = Asset('eip155:42161/erc20:0x82af49447d8a07e3bd95bd0d56f35241523fbab1')
+A_WETH_ARB = Asset('eip155:42161/erc20:0x82aF49447D8a07e3bd95BD0d56f35241523fBab1')
```

Comment:
ups big fail. How was it not noticed, where is it used?

Reply:
Not sure, it's mainly used in the utils section, and decoding gmx and weth.

Reply:
It's because it's used in the test and the tests do equality check.

```
>>> from rotkehlchen.assets.asset import Asset
>>> Asset('eip155:42161/erc20:0x82aF49447D8a07e3bd95BD0d56f35241523fBab1') == Asset('eip155:42161/erc20:0x82af49447d8a07e3bd95bd0d56f35241523fbab1')
True
>>> 
```

And that check is case insensitive.